dask/dataframe/tests/test_dataframe.py

```diff
@@ -2423,6 +2423,22 @@ def test_dataframe_itertuples():
         assert a == b
 
 
+def test_dataframe_itertuples_with_index_false():
+    df = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [10, 20, 30, 40]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    for (a, b) in zip(df.itertuples(index=False), ddf.itertuples(index=False)):
+        assert a == b
+
+
+def test_dataframe_itertuples_with_name_none():
+    df = pd.DataFrame({'x': [1, 2, 3, 4], 'y': [10, 20, 30, 40]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    for (a, b) in zip(df.itertuples(name=None), ddf.itertuples(name=None)):
```

Comment:
IIRC, plain tuples compare equal to namedtuples. Can you add an additional assertion that `type(a) is type(b)`?